### ISSUE TITLE
[FEATURE] Use Tear Away Pants against flaming leaves

### ISSUE THREAD

**Issue by Tapionator** [loathers/garbage-collector#2101]
I noticed this was just supported for the pirahna plant. Would it make sense to use them also against the flaming leaves enemies to help gain some extra advs?

**Comment by Tapionator** [comment]
Also something to consider, using free runs to try to obtain additional adventures if you have Spring Beach charter Fun-Guy Mansion or The Fungal Nethers available with the bander/GAP/Naval ring in conjunction with the tearaway pants. I noticed my Flame6 alt doesn't use any of it's free runs from it's soup enhanced bander. (Would of though it would leverage Shadow Rift buffs). 
[Flame6_20240913.txt](https://github.com/user-attachments/files/17000325/Flame6_20240913.txt)

**Comment by horrible-little-slime** [comment]
Did a quick search, those are precisely the only two relevant zones we can use this as a free run target for:
```
> js Location.all().filter((l) =>getMonsters(l).length && getMonsters(l).every((m) => m.phylum === Phylum.get("plant")) && l.combatPercent === 100)

Returned: aggregate location [11]
0 => Your Mushroom Garden
1 => The Hedge Maze
2 => An Overgrown Shrine (Northwest)
3 => An Overgrown Shrine (Southwest)
4 => An Overgrown Shrine (Northeast)
5 => An Overgrown Shrine (Southeast)
6 => The Fungal Nethers
7 => Site Alpha Greenhouse
8 => Shivering Timbers
9 => The Fun-Guy Mansion
10 => The Druidic Campsite
```
The others are illegitimate for a variety of reasons

**Comment by Tapionator** [comment]
Yeah wish there were more areas but those were the only 2 I could think of. Players who are not ascending will definitely benefit from the nethers being open and others with the spring break charter will probably appreciate a zone being used for a handful of extra advs (depending on what they have of course). I would assume a new calculation would need to be added for value of the free run depending on the adv gained from the pants (unsure if there is a current pref for terawaypants advs gained yet.
